packages/std/src/results/submessages.rs

```diff
@@ -67,6 +67,11 @@ impl<T> SubMsg<T> {
         Self::reply_on(msg.into(), id, ReplyOn::Always)
     }
 
+    /// create a `SubMsg` that will never `reply`. This is equivalent to standard message semantics.
+    pub fn reply_never(msg: impl Into<CosmosMsg<T>>, id: u64) -> Self {
```

Comment:
I think the `new` constructor could use this function internally now for brevity.
I am wondering a bit about the `id` though. Does it really make sense to provide an `id` for a message you never get a reply for?

Reply:
I'd internalize the id and use `UNUSED_MSG_ID` just like `new` does

Reply:
agreed